source/frontend/src/features/leases/shared/propertyPicker/LeasePropertySelector.tsx

```diff
@@ -137,14 +177,15 @@ export const LeasePropertySelector: React.FunctionComponent<
                 </Col>
               </Row>
               <Section header="Selected properties">
+                {propertyIndexToRemove}
```

Comment:
was this here for debugging? or is the story calling for showing the 0-based index of the property on the ui?